public/electron.js

```diff
@@ -101,7 +119,7 @@ function createWindow() {
       accelerator: 'CommandOrControl+O',
     },
     {
-      label: "Recharger l'application",
+      label: `Recharger l'application`,
```

Comment:
Change ` in '

Reply:
It's for escaping the ' inside the string